lightningd/lightningd.c

```diff
@@ -133,6 +135,28 @@ static void test_daemons(const struct lightningd *ld)
 	}
 	tal_free(ctx);
 }
+/* Check if all the daemons exist in the specified directory. */
+static bool has_all_daemons(const char* daemon_dir)
+{
+	size_t i;
+	bool retval;
```

Comment:
Minor stylistic nit:  I try to give vars meaningful names, in this case, I'd probably use something like "daemon_missing = false".

Reply:
I agree, that would indeed be better.